src/app/component/explorer/index.vue

```diff
@@ -253,17 +255,52 @@ export default {
         },
 
         hasRowActive() {
-            return !! this.rowActive
+            return !! this.rowActive;
         },
     },
 
     methods: {
+        resizeTable () {
+            let title, startOffset;
+
+            Array.prototype.forEach.call(
+                document.querySelectorAll("table th"),
```

Comment:
`NodeList` object has the `forEach` method, so call directly. `document.querySelectorAll().forEach()`

https://developer.mozilla.org/en-US/docs/Web/API/NodeList/forEach

Reply:
Another thing, use `v-ref`